docker/builder/build-tools.sh

```diff
@@ -4,17 +4,24 @@
 set -e
 
 PROFILE=cli
+# TODO use an unstripped cli profile, we dont need to use release image for
+# forge cli, that might save build time / space without losing backtrace
+FORGE_PROFILE=release
 
 echo "Building tools and services docker images"
 echo "PROFILE: $PROFILE"
+echo "FORGE_PROFILE: $PROFILE"
 echo "CARGO_TARGET_DIR: $CARGO_TARGET_DIR"
 
+cargo build --locked --profile=$FORGE_PROFILE \
+    -p aptos-forge-cli \
+    "$@"
```

Comment:
Build takes 16m now. :(

Reply:
Hmmmmm I wonder if i can just run the builds in parallel :P

Reply:
Alternatively do we actually need forge in the tools image?

Reply:
I think we can just add forge to aptos-node builder. Even if we inherit the features/release flags from aptos-node, I think its fine with the forge-cli. We dont use those features anyhow. That way we don't need additional parallel builder. wdyt?

Once we build a binary, it doesn't matter what binary goes where, we can just copy from the build step to multiple images. We could have a forge specific image and also include it in tools